sdk/search/Azure.Search.Documents/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 11.4.0 (Unreleased)
+## 11.4.0 (2022-09-06)
```

Comment:
Should this be 9/7 since Monday is a holiday?

Reply:
September 6th is on Tuesday.

Reply:
Understood, but we normally push the release schedule for a holiday... but I just found the dates announcement and it looks like Tuesday is still the day.   Ignore me.  😄